packages/react-i18n/src/i18n.ts

```diff
@@ -204,11 +204,15 @@ export class I18n {
   }
 
   translationKeyExists(id: string) {
+    const errorHandler = this.onError;
     try {
+      (this.onError as any) = defaultOnError;
```

Comment:
This property should use the `private` modifier instead of being casted

Reply:
How do you do that? :)

Reply:
I think Chris and I submitted our comments around the same time so my point isn't valid now, but the reason you had to cast the property is because `onError` was declared using the `readonly` modifier so you couldn't reassign this member without tricking the compiler. Changing the modifier to `private` would have allowed reassignment without the cast. [More info](https://www.typescriptlang.org/docs/handbook/classes.html#public-private-and-protected-modifiers)

Reply:
Cool, thanks for the details